### DISCUSSION TITLE
Checks are empty!

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#5864] [Q&A]
Hi,

I've installed Weblate on Debian. All is working except from checks. 

URL /checks/ has an empty table. 

I tried forcing checks on translations but no luck.

**weblate list_checks** correctly lists them. 

**weblate check --deploy** shows no critical warning (only something about git which is irrelevant).

```
WARNINGS:
?: (security.W018) You should not have DEBUG set to True in deployment.
?: (weblate.W033.Gerrit) Failure in loading VCS module for Gerrit: git: 'review' is not a git command. See 'git --help'.
(1)     HINT: https://docs.weblate.org/en/weblate-4.5.3/vcs.html
?: (weblate.W033.Subversion) Failure in loading VCS module for Subversion: git: 'svn' is not a git command. See 'git --help'.

The most similar command is
        serve
(1)
        HINT: https://docs.weblate.org/en/weblate-4.5.3/vcs.html

INFOS:
?: (weblate.I021) Error collection is not set up, it is highly recommended for production use
        HINT: https://docs.weblate.org/en/weblate-4.5.3/admin/install.html#collecting-errors
?: (weblate.I028) Backups are not configured, it is highly recommended for production use
        HINT: https://docs.weblate.org/en/weblate-4.5.3/admin/backup.html
```

in **settings.py** all checks are  enabled. I have spent so many hours trying different things but nothing seems to be working! :( 

I haven't managed to find anything in any logs that might indicate what's wrong. 

Any help appreciated!

**Comment by nijel** [comment]
> URL /checks/ has an empty table.

Lists  failing quality checks, seems like there are none failures.

> weblate list_checks correctly lists them.

Lists installed quality checks.

> weblate check --deploy shows no critical warning (only something about git which is irrelevant).

Shows Django system checks, which is something entirely different.

**Reply by ghubz** [reply]
Thanks for your prompt response. I have another test install in docker that has the exact same translation files uploaded and it shows many check warnings. In this new installation though it doesn't. It's the exact same file and setup. The strings/languages are imported successfully. 

See attached 

![image](https://user-images.githubusercontent.com/82657457/115029800-05f08600-9ecf-11eb-8cf9-1a358157f2d8.png)

**Reply by ghubz** [reply]
Does it matter than in the second case it is being added as Glossary?

**Reply by nijel** [reply]
Yes, glossary has currently all checks disabled.

**Reply by ghubz** [reply]
Unless check-glossary flag is manually added correct? 
Thanks for your help btw

**Reply by nijel** [reply]
That one enables checking translations against glossaries.
